.github/workflows/build-images-ci.yml

```diff
@@ -59,17 +59,26 @@ jobs:
             echo "tag=${{ github.sha }}" >> $GITHUB_OUTPUT
           fi
 
-      - name: Checkout Source Code
+      - name: Checkout master branch
```

Comment:
nit
```suggestion
      - name: Checkout main branch
```

Reply:
Ah nice catch. PR's already merged, but let's fix this in a follow-up

Reply:
https://github.com/cilium/tetragon/pull/1613